Library/Homebrew/utils/shell.rb

```diff
@@ -0,0 +1,64 @@
+require 'shellwords'
```

Comment:
Please always use double quotes for strings. More importantly: The `shellwords` module doesn't exist in Ruby 1.8.7, a version of Ruby we still support, though we're in the process of phasing out that support. This is the main reason why tests on our OS 10.9 test bot fail. I can think of two solutions to this problem:

1. Only use `shellwords` with Ruby 2.0 and above, fall back to not quoting (like before) for older Rubies.
2. Re-implement the required functionality, like you already did with `csh_quote`, and drop the `shellwords` dependency completely (if that's feasible).